### ISSUE TITLE
False Positive in file-name-casing's kebab-case option for file names with periods

### ISSUE THREAD

**Issue by agustinbcu01** [palantir/tslint#4324]
### Bug Report

Angular CLI: 7.1.0
Node: 10.13.0
OS: win32 x64
Angular: 7.1.0
_TSLint version__: 5.11
- __TypeScript version__: 3.1.6
- __Running TSLint via__: CLI

file name is in proper case 

with `tslint.json` configuration:
"file-name-casing": [true, "kebab-case"]

#### Actual behavior
reports 
```
Subscriptions.Service.ts[1, 1]: File name must be kebab-case
```
but the file real name is  `subscriptions.service.ts`
#### Expected behavior
pass the lint

**Comment by JoshuaKGoldberg** [comment]
Thanks for this @agustinbcu01, good find!

It looks like the `file-name-casing` rule's missing tests for the `kebab-case` option. Interesting.

**Comment by isabelleflores** [comment]
I can't seem to reproduce this issue. When I try this with similarly formatted filenames, I don't get any tslint errors.

**Comment by agustinbcu01** [comment]
You can close them. Yesterday i did some packages updates and start working fine.

**Comment by JoshuaKGoldberg** [comment]
Awesome, thanks!
